haiku-apps/beostilink/beostilink-0_git.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="A port of GTKTILink for BeOS"
+DESCRIPTION="BeOSTiLink is the port of GTKTILink on BeOS, for connecting to Texas Instruments calculators."
+HOMEPAGE="https://github.com/HaikuArchives/BeOSTiLink"
+COPYRIGHT="Severin alias Surfkiller"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="5311055c86059ae95d551aaac4e982c6bac5c4cd"
+SOURCE_URI="https://github.com/HaikuArchives/BeOSTiLink/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="3c6d33c94cc189cf6f8c50f11a0e1580f54e8df91cec0a8478340bc81f276783"
+SOURCE_FILENAME="BeOSTiLink-0~$srcGitRev.tar.gz"
+SOURCE_DIR="BeOSTiLink-$srcGitRev"
+#PATCHES=""
+
+ARCHITECTURES="x86_gcc2"
+
+PROVIDES="
+	beostilink = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREQUIRES="
+	makefile_engine
+	cmd:autoconf
+	cmd:gcc
+	cmd:g++
+	cmd:make
+        "
+
+BUILD()
+{
+	make -C src
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir/BeOSTILink
+	cp -r src/* $appsDir/BeOSTILink
+
+	addAppDeskbarSymlink $appsDir/BeOSTILink/objects.x86-cc2-debug/BeOSTiLink
```

Comment:
This is not good. We dont need useless subfolders here. Please rework the Directory structure, eliminate the unneeded source and object Files, make it clean and nice please.

Reply:
@extrowerk is correct here, objects.x86-cc2-debug is your OBJDIR folder created on build ... it should not be in the package ...